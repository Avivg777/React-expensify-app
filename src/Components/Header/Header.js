import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = () => (
    <header>
        <h1>Expensify</h1>
        <NavLink to="/" activeClassName="active" exact={true}>Home</NavLink>
        <NavLink to="/create" activeClassName="active">Add Expense Page</NavLink>
        <NavLink to="/edit" activeClassName="active">Edit Expense Page</NavLink>
        <NavLink to="/help" activeClassName="active">Help</NavLink>
    </header>
);

export default Header;