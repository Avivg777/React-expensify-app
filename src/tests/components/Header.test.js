import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../Components/Header/Header';

test('Should render Header correctly', () => {
    // const renderer = new ReactShallowRenderer();
    // renderer.render(<Header />);
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
    const wrapper = shallow(<Header />);

    expect(wrapper).toMatchSnapshot();
});