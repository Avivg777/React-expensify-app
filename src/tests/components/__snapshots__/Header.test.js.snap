// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Header correctly 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="active"
    ariaCurrent="true"
    exact={true}
    to="/"
  >
    Home
  </NavLink>
  <NavLink
    activeClassName="active"
    ariaCurrent="true"
    to="/create"
  >
    Add Expense Page
  </NavLink>
  <NavLink
    activeClassName="active"
    ariaCurrent="true"
    to="/edit"
  >
    Edit Expense Page
  </NavLink>
  <NavLink
    activeClassName="active"
    ariaCurrent="true"
    to="/help"
  >
    Help
  </NavLink>
</header>
`;
